app/helpers/samples_reports_helper.rb

```diff
@@ -0,0 +1,52 @@
+module SamplesReportsHelper
+  def get_rates(samples, signal)
+    confusion_matrix = confusion_matrix(samples, signal)
+
+    if confusion_matrix[:actual_positive] > 0 && confusion_matrix[:actual_negative] > 0 
+      tpr=confusion_matrix[:true_positive].to_f/confusion_matrix[:actual_positive].to_f
+      fpr =confusion_matrix[:false_positive].to_f/confusion_matrix[:actual_negative].to_f
+      [fpr, tpr]
+    else
+      [0,0]
```

Comment:
This is returning an Array<Integer> while the other branch returns an Array<Float>, let's always return floats?
```suggestion
      [0.0, 0.0]
```